src/displayapp/widgets/PopupMessage.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <lvgl/lvgl.h>
+
+namespace Pinetime {
+  namespace Applications {
+    namespace Widgets {
+      class PopupMessage {
+      public:
+        PopupMessage(const char* msg);
```

Comment:
please add a note explaining that the string isn't copied, meaning the pointed to memory must be alive for the popup message to still work